features/auth/signup/BasicForm.tsx

```diff
@@ -116,6 +103,26 @@ export default function BasicForm({
           helperText={errors?.email?.message ?? " "}
           error={!!errors?.email?.message}
         />
+        <InputLabel className={authClasses.formLabel} htmlFor="password">
+          Password
```

Comment:
This label is still hardcoded, please update it to retrieve with the `t` function instead!